Sources/RichTextKit/Actions/RichTextAction.swift

```diff
@@ -100,6 +149,29 @@ public extension RichTextAction {
         case .stepSuperscript(let steps): .actionStepSuperscript(steps)
         case .toggleStyle(let style): style.titleKey
         case .undoLatestChange: .actionUndoLatestChange
+        case .backgroundColor: .backgroundColor
```

Comment:
When we have a single setColor, these properties can just check the color's icon and title key.